Packs/CortexDataLake/Integrations/CortexDataLake/CortexDataLake.yml

```diff
@@ -25,6 +25,16 @@ configuration:
   name: first_fetch_timestamp
   required: false
   type: 0
+- additionalinfo: Choose from what table incidents will be fetched from.
+  defaultvalue: firewall.threat
+  display: Fetch Table
+  hidden: false
+  name: fetch_table
+  options:
+  - firewall.threat
+  - firewall.file_data
```

Comment:
do we want to support fetching from both tables ?

Reply:
Simultaneously? there was no decision about it, but it'll be a bit problematic as there is no way to know from what table the incident was fetched once it's fetched

Reply:
You are right @DeanArbel, maybe in that case we will not allow fields selection to simplify the fetch.
@prasenshelar do we want to support fetch Simultaneously?
or will the user have to have 2 instances for that?

Reply:
@bakatzir we will have to decide this. We can ask users to create separate instances or just keep it in one.
The requirement is to be able to fetch entries from both tables and keep it simple as possible. 

If we can just add a parameter to the entries that says from which table the log was fetched from then that would be great.